<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowCities</class>
 <widget class="QMainWindow" name="ShowCities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vis innlagte byer og steder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Her kan du se stedene som er lagt inn i databasen. Vil du bytte ID-en til et sted, mÃ¥ dette gjÃ¸res via kommandolinja.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelCityID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>StedID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCityID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stedsnavn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCityName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCountryID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>LandID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCountryID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCountryName">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>$countryName</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirst">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
